### ISSUE TITLE
 duckdb_prepare_R: Unknown column type for prepare: LIST<VARCHAR>

### ISSUE THREAD

**Issue by cboettig** [duckdb/duckdb#2278]
**What does happen?**
In attempting to read an existing Parquet file, I see the error 

**What should happen?**
duckdb should successfully read the file (or perhaps indicate the parquet file contains unimplemented features)

**To Reproduce**

```r
download.file("https://minio.cirrus.carlboettiger.info/shared-data/occurrence.parquet/000000", "part0.parquet") # 109 MB
conn <- DBI::dbConnect(duckdb::duckdb())
DBI::dbSendQuery(conn, paste0("SELECT * FROM 'part0.parquet';"))
```
From what I can tell, this only impacts the R client though.  The same commands on the CLI client seem to be working fine.  

**Environment (please complete the following information):**
 - OS:  Linux; R client
 - DuckDB Version (GitHub master branch as of now)

**Before submitting**
- [x] Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?
- [x] Have you tried this on the latest `master` branch? In case you cannot compile, you may find some binaries here: https://github.com/duckdb/duckdb/releases/tag/master-builds

**Comment by hannesmuehleisen** [comment]
Yes, the R client is still missing support for lists and structs. Will add those soon though.

**Comment by hannesmuehleisen** [comment]
With the linked PR this file is now readable

**Comment by cboettig** [comment]
:tada: works great!
